tools/gen_compilation_database.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+
+import argparse
+import os
+import json
+
+def generateCompilationDatabase():
+  os.system("bazel/gen_compilation_database.sh //source/... //test/... //tools/...")
```

Comment:
FYI this will break when folks try to use this in repos that consume Envoy, but we can check this in and iterate as needed.

Reply:
ack